Makefile

```diff
@@ -12,36 +7,37 @@ MANPREFIX = $(PREFIX)/share/man
 DOCPREFIX = $(PREFIX)/share/doc/nsxiv
 
 # autoreload backend: inotify/nop
-AUTORELOAD = inotify
+AUTORELOAD ?= inotify
```

Comment:
Should we take this opportunity to change `AUTORELOAD` to `HAVE_INOTIFY` for consistency?

Reply:
How would we implement it? I was thinking about using preprocessor ifdefs but decided to drop that idea because that would mean if someone changes AUTORELOAD it wont trigger a rebuild.

Reply:
I don't think changes to makeflags should trigger a rebuild just like enabling/disabling libs won't trigger a rebuild. I think ifdefs are fine and to compliment that maybe we should just merge `autoreload_inofity` and `autoreload_nop` instead of having reverse ifdefs in each

Reply:
I guess we can do what we're doing with the opt libs. Very hacky, but since `ifeq` isn't posix, not much else I can think of.

Reply:
Fixed in https://github.com/nsxiv/nsxiv/pull/71/commits/12c0881de16dec2eb8d7474825bbf70c9d021f63